spec/services/job_posting_poller_service_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe JobPostingPollerService, :vcr do
+  around do |spec|
+    ClimateControl.modify(SMARTJOBBOARD_PUBLIC_URL: 'https://www.myzivi.ch') do
+      spec.run
+    end
+  end
+
+  describe '.poll' do
+    subject(:polled_postings) { described_class.poll }
+
+    # cassette manually edited
+    it 'polls the most recent jobs' do
+      expect { polled_postings }.to change(JobPosting, :count).by(2)
+
+      expect(polled_postings).to contain_exactly(
+        have_attributes(title: 'Naturschutzeinsatz'),
```

Comment:
How do you know this will always be true?

Reply:
Because I used vcr